### DISCUSSION TITLE
怎么全局模式

### DISCUSSION THREAD

**Discussion by rainricky** [SagerNet/SagerNet#352] [Q&A]
请教大家怎么实现全局模式，具体该怎么设置?请大家帮帮忙，谢谢！

**Comment by Nelocompne** [comment]
实现所谓的全局路由有两种方式：

- 在**路由页面**中关闭所有的路由规则
- 在**设置页面**的**路由设置**下的**分应用代理**中选择代理并指定所有应用

**Comment by mailshuxin** [comment]
在route里面加一条规则，端口，0-65535，bypass，放最后
